test/test.dart

```diff
@@ -623,8 +629,8 @@ Future<String> createServerApiKey(App app, String name, {bool enabled = true}) a
   return await client.createApiKey(baasApp, name, enabled);
 }
 
-Future<Realm> getIntegrationRealm({App? app, ObjectId? differentiator}) async {
-  app ??= App(await getAppConfig());
+Future<Realm> getIntegrationRealm(AppConfiguration appConfiguration, {App? app, ObjectId? differentiator}) async {
```

Comment:
Its weird that we need to pass appConfig as required but also we can pass the full app as optional. From the caller Its not clear enough what this method does with the arguments supplied.

Reply:
I agree that this refactoring is not needed anymore, since I changed the idea that I had.